desktop/src/gui/preferences_dialog.rs

```diff
@@ -320,14 +314,29 @@ fn filename_pattern_name(locale: &LanguageIdentifier, pattern: FilenamePattern)
     }
 }
 
-fn backend_availability(descriptors: &Descriptors, backend: wgpu::Backends) -> wgpu::Backends {
-    if descriptors
-        .wgpu_instance
-        .enumerate_adapters(backend)
-        .is_empty()
-    {
+fn backend_availability(instance: &wgpu::Instance, backend: wgpu::Backends) -> wgpu::Backends {
+    if instance.enumerate_adapters(backend).is_empty() {
         wgpu::Backends::empty()
     } else {
         backend
     }
 }
+
+fn find_available_graphics_backends() -> wgpu::Backends {
```

Comment:
earlier you mentioned that

> I think we can remove that code now, wgpu added an api to show which backends are (roughly) available

Just wanna make sure whether that was checked :D

Reply:
Yeah, turns out that's unreleased still :(
`wgpu::Instance::enabled_backend_features()` in the next major wgpu update